Gems/SceneProcessing/Code/Source/Generation/Components/MeshOptimizer/MeshOptimizerComponent.cpp

```diff
@@ -384,6 +384,16 @@ namespace AZ::SceneGenerationComponents
 
             for (const IMeshGroup& meshGroup : meshGroups)
             {
+                if (!selectedNodes.contains(&meshGroup))
+                {
+                    AZ_Warning(
+                        AZ::SceneAPI::Utilities::LogWindow,
+                        false,
+                        "MeshGroup %s wasn't found in the list of selected nodes.",
+                        meshGroup.GetName().c_str());
```

Comment:
Do all mesh group have a non-null name?

Reply:
AFAIK yes? I don't _think_ they can even have empty non-null names, since all of the node hierarchy structures are based on concatenated names, but I'm not 100% sure of that.